### ISSUE TITLE
Crashes on Menu Click (10.7)

### ISSUE THREAD

**Issue by ghost** [markbao/issuepost-issues#1]
I run the application, menubar icon shows, when I click on the menubar icon, it hangs (stays highlighted for a second), then crashes (closes with no notification). 

(os x 10.7.5 here)

**Comment by markbao** [comment]
Dang, that's weird. Thanks for reporting this. Could you go to **Finder**, go to **Go > Go to File...**, type in `~/Library/Logs/DiagnosticReports/`, and see if there's an entry starting with `Issuepost` there?

**Comment by ghost** [comment]
Yep - here's the newest crash report for Issuepost: http://pastebin.com/vAuiiM1w  
(I tried three times, same error, each report appears to be practically the same)

**Comment by markbao** [comment]
Thanks for this. Seems like 10.7 has multiple problems that will make it a pain to support :/
